docker-compose-dev.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.3" 
+
+services:
+  devapp:
+    container_name: todo-app-dev
+    build:      
+      context: .
+      dockerfile: Dockerfile
+      target: development
+      args: 
+        - PYTHON_VERSION=3.9-slim-buster
+    image: todo-app-dev:latest
+    volumes: 
+      - type: bind
+        source: ${PROJECT_FOLDER}
+        target: /app
+    ports:
+      - 5000:5000
+    entrypoint: poetry run flask run --host 0.0.0.0
```

Comment:
These entrypoints are primarily to override the default entrypoints specified in the Dockerfile - this would be better placed as the Dockerfile's entrypoint so that images produced with that file are more intuitive to run (without needing to also find an appropriate docker-compose)